shared/common-adapters/video.desktop.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+import * as React from 'react'
+import Measure from 'react-measure'
+import type {Props, State} from './video'
+import * as Styles from '../styles'
+import {getVideoSize} from './video.shared'
+import logger from '../logger'
+
+export default class extends React.PureComponent<Props, State> {
+  state = {
+    containerHeight: 0,
+    containerWidth: 0,
+    loadedVideoSize: false,
+    videoHeight: 0,
+    videoWidth: 0,
+  }
+
+  _mounted = false
+
+  _onContainerResize = ({bounds}) =>
```

Comment:
likely want to check _mounted here too, i don't remember if that can get called in an unsafe way, better safe than sorry